fault/__init__.py

```diff
@@ -1,8 +1,12 @@
+from .real_type import RealIn, RealOut
+from .elect_type import ElectIn, ElectOut
 from .tester import Tester
 from .power_tester import PowerTester
-from .value import Value, AnyValue, UnknownValue
+from .value import Value, AnyValue, UnknownValue, HiZ
 import fault.random
 from .symbolic_tester import SymbolicTester
+from .verilogams import VAMSWrap
+from .tester_samples import *
```

Comment:
Can we change this from an `import *` to explicitly importing what you want? In practice, I prefer the style of explicit imports as it's easier to track where things are coming from (avoids bugs where something is accidentally imported when it wasn't meant to be, possibly clobbering an existing name)

Reply:
Sorry about that -- will fix it in this week's PR.